### PR TITLE
Inverted radial progress drawing

### PR DESCRIPTION
**Pull request by rzimmer**
Thank you for the great library! We had the requirement to invert the way the progress is drawing in radial mode, so here is a corresponding pull request of a working version. Hopefully you can include this upstream. Feel free to comment on this request.

More pull requests may follow...

### CODE REVIEW

**Comment by rzimmer** [comment]
As it was so easy, I just added a counterclockwise drawing mode which works for both the normal and the inverted mode.

File: library/src/main/java/com/filippudak/ProgressPieView/ProgressPieView.java [hunk 1]
```diff
@@ -318,6 +328,21 @@ public void setStartAngle(int startAngle) {
...
+
+    /**
+     * Sets the inverted state.
+     * @param inverted draw the progress inverted or not
+     */
+    public void setInverted(boolean inverted) {
+        mInverted = inverted;
+    }
+
+    /**
```
Comment (FilipPudak) [inline library/src/main/java/com/filippudak/ProgressPieView/ProgressPieView.java:345]: Missing the counterClockwise getter/setter.

File: README.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ Choose from the broad spectre of styleable elements:
  * `strokeColor` - The color of stroke around the view.
  * `backgroundColor` - The color of the views background.
  * `progressColor` - The color view of the views progress.
+ * `inverted` - Inverts the drawing of progress (fill radial only).
```
Comment (FilipPudak) [inline README.md:14]: missing the counterClockwise here.

**Comment by rzimmer** [comment]
Sorry, I forgot about this, you are right. I just added another commit with the mentioned changes.

**Pull request merged**